### ISSUE TITLE
SSO problem when there is restrictions in the cookies configuration (incognito mode)

### ISSUE THREAD

**Issue by Tansito** [Qiskit/saiba#329]
### What is the current behavior?

Directus makes use of cookies to share information between the front and the backend. Due to this, configurations with restricted cookies management create problems in the singin workflow, for example.

### Additional information
Here is the configuration followed to setup the authentication: https://docs.directus.io/self-hosted/sso.html#seamless-sso

**Comment by Tansito** [comment]
@abdonrd I will need your help to review this. Let's see if we can add it in this sprint or we will need to leave for the next one.

**Comment by abdonrd** [comment]
Sure, let's investigate about it!

**Comment by yoshiemuranaka** [comment]
@Tansito I think browser cookies may need to be enabled for incognito mode for the SSO login flow to work. would this resolve this issue or is there something else that should be investigated?

**Comment by Tansito** [comment]
I don't know what you think @yoshiemuranaka , @vabarbosa , @abdonrd but maybe we reached the time when we need to add the cookies policy control from IBM:

<img width="320" alt="Screenshot 2022-12-01 at 09 59 31" src="https://user-images.githubusercontent.com/9059044/205010172-da501e99-8285-40ce-8e9c-5b942f32a065.png">

**Comment by abdonrd** [comment]
As I mention here (https://github.com/Qiskit/saiba/issues/321#issuecomment-1333509015), I assume the cookies have to be enabled.

**Comment by Tansito** [comment]
Yes, I assume that behaviour is something we can fix adding the cookie script (I think).

**Comment by vabarbosa** [comment]
yes, definitely worth investigating the cookie consent solution from IBM

**Comment by yoshiemuranaka** [comment]
The IBM cookie consent solution does not solve the SSO issue, but after internal discussions, we assume that the set-cookie header is blocked on Safari because the cookie domain attribute is different from the deployed staging domain. Our assumption is that if the UI is deployed to a `qiskit.org` subdomain, the backend cookie will be treated as first-party and would _**not**_ be blocked on Safari

https://support.apple.com/guide/safari/prevent-cross-site-tracking-sfri40732/mac
